cmd/aws-vpc-cni/main.go

```diff
@@ -292,23 +295,41 @@ func generateJSON(jsonFile string, outFile string, getPrimaryIP func(ipv4 bool)
 
 	byteValue = []byte(netconf)
 
+	// Unmarshall current conflist into data in order to chain any requested plugins
+	data := NetConfList{}
```

Comment:
While, I'm Ok with this flow. How about we do this only if either of the cases is true. We're anyways deriving both the env vars and we can use them to decide if we even need to do this..

```
enBandwidthPlugin := utils.GetBoolAsStringEnvVar(envEnBandwidthPlugin, defaultEnBandwidthPlugin)
disableContainerV6 := utils.GetBoolAsStringEnvVar(envDisableContainerV6, defaultDisableContainerV6)

if enBandwidthPlugin || disableContainerV6{
    //Unmarshall the conflist file
}
```

I understand that you might need another set of `if` checks inside to decide which one to chain but considering these env vars are probably used by <1% of Cx, I'm thinking it's alright.

Reply:
Yeah, I figured one extra JSON unmarshall and marshall was fine, but I can avoid this